### PR TITLE
Makes the Cosmic Skull glow purple when it has uses

### PR DESCRIPTION
**Pull request by StrangeWeirdKitten**
## About The Pull Request

Per Title

## Why It's Good For The Game

I think the loot should glow purple. It would be pretty cool 😄  Just a little added flavour to the item.

image

## Changelog

:cl:
add: Cosmic Skull glows purple.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/voidwalker/voidwalker_loot.dm [hunk 1]
```diff
@@ -37,3 +40,4 @@
 	uses--
 	if(uses <= 0 )
 		icon_state = drained_icon_state
+		light_on = 0
```
Comment (Jacquerel) [inline code/modules/antagonists/voidwalker/voidwalker_loot.dm:43]: should be using `TRUE` and `FALSE` here if this is treated just as a boolean

Comment (StrangeWeirdKitten) [reply code/modules/antagonists/voidwalker/voidwalker_loot.dm]: yep

**Comment by Jacquerel** [comment]
On uh, what basis should this not have GBP applied to it?

**Comment by StrangeWeirdKitten** [comment]
> On uh, what basis should this not have GBP applied to it?

I was unsure what I would throw it under the changelog as. But if you want GBP to be applied, then that's fine.

**Comment by necromanceranne** [comment]
We use GBP: No Update for people to be able to hotfix/readd something missed in a recently merged PR so that they're not being double-charged/doulbe-rewarded for this. Though PRs like this do really make me wish we had tweak still. It's not really a feature, it's just kind of set dressing.

**Comment by Jacquerel** [comment]
> We use GBP: No Update for people to be able to hotfix/readd something missed in a recently merged PR so that they're not being double-charged/doulbe-rewarded for this. Though PRs like this do really make me wish we had tweak still. It's not really a feature, it's just kind of set dressing.

I do also think it's a _bit_ excessive to have the full "addition" alteration for this change but it's also not technically quality of life or an "image addition" 😔

**Pull request merged**